src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -347,21 +347,35 @@ internal IOperation CreateInternal(BoundNode boundNode)
                         }
                     }
 
-                    return new CSharpLazyNoneOperation(this, boundNode, _semanticModel, boundNode.Syntax, constantValue, isImplicit: isImplicit, type: null);
+                    ImmutableArray<IOperation> children = GetIOperationChildren(boundNode);
+                    return new NoneOperation(children, _semanticModel, boundNode.Syntax, type: null, constantValue, isImplicit: isImplicit);
 
                 default:
                     throw ExceptionUtilities.UnexpectedValue(boundNode.Kind);
             }
         }
 
+#nullable enable
```

Comment:
nit: can we remove the disable before previous method instead?

Reply:
The next PR will fully nullable-enable all these files.